web/themes/custom/liibarrio2020/scss/layout/_hero.scss

```diff
@@ -16,7 +16,7 @@
   }
 
   &__content {
-    height: 0; //height value needed for IE11
+    height: 100; //height value needed for IE11
     min-height: 21.875rem; //350px
     padding: 0 $spacer-lg;
     display: flex;
```

Comment:
```suggestion
    padding: 0 $spacer-lg;
```

Reply:
This works for me: the content will now push the container to fill the vertical space.